### PR TITLE
WIP: Experiment with language around Consul's connect subsystem

### PR DESCRIPTION
**Pull request by jkirschner-hashicorp**

### CODE REVIEW

**Comment by david-yu** [comment]
Nice I like what I see here @jkirschner-hashicorp

File: website/content/commands/connect/expose.mdx [hunk 1]
```diff
@@ -20,8 +20,8 @@ about Ingress gateways.
 ```text
 Usage: consul connect expose [options]
 
-  Exposes a Connect-enabled service through the given ingress gateway, using the
-  given protocol and port.
+  Exposes a service in the service meshthrough the given ingress gateway,
```
Comment (johncowen) [inline website/content/commands/connect/expose.mdx:23]:
```suggestion
  Exposes a service in the service mesh through the given ingress gateway,
```

**Review by johncowen — COMMENTED** [review]
Was driving by and spotted a tinsy typo

**Comment by david-yu** [comment]
Could we re-open, re-base and get feedback on just the portions that require no code changes?

**Comment by jkirschner-hashicorp** [comment]
@david-yu : I need to get alignment on approach internally before applying more broadly. I can use what I have shown here to get feedback. I'll take this up in a few weeks.

**Comment by jkirschner-hashicorp** [comment]
Replaced by https://github.com/hashicorp/consul/pull/17222

**Pull request closed without merging**